### PR TITLE
redo the pull request process

### PR DESCRIPTION
**Pull request by yfzhaosteve**
Do the pull request again so I can remember the process better.

### CODE REVIEW

File: yfzhaosteve-file.md [hunk 1]
```diff
@@ -3,4 +3,4 @@
 * item1
 * item2
```
Comment (fggrimmer) [inline yfzhaosteve-file.md:4]: @yfzhaosteve  item 2 comment. What is this for? :+1:

Comment (jmatracia) [inline yfzhaosteve-file.md:4]: are you sure :100:

File: yfzhaosteve-file.md [hunk 2]
```diff
@@ -3,4 +3,4 @@
...
 
-check1
+check2
```
Comment (pkirby) [inline yfzhaosteve-file.md:6]: :tada: 

## Commented

File: yfzhaosteve-file.md (hunk 1 above)
Comment (yfzhaosteve) [inline yfzhaosteve-file.md:4]: @fggrimmer This is just for fun! :-)

**Comment by yfzhaosteve** [comment]
:shipit: :ship:

**Pull request merged**